test/route_patterns/repo_test.exs

```diff
@@ -1,50 +1,84 @@
 defmodule RoutePatterns.RepoTest do
   use ExUnit.Case, async: true
-  @moduletag :external
 
-  alias RoutePatterns.{Repo, RoutePattern}
+  import Mox
+  import Test.Support.Factory.MbtaApi
+
+  alias RoutePatterns.RoutePattern
+
+  setup :verify_on_exit!
 
   describe "get" do
     test "returns a single route pattern" do
-      assert %RoutePattern{id: "111-5-0"} = Repo.get("111-5-0")
+      expect(MBTA.Api.Mock, :get_json, fn url, _ ->
+        assert url == "/route_patterns/111-5-0"
```

Comment:
Is there something magical about this route pattern number? If not, we should use a random one like the test below.